libs/payments/cart/src/lib/types.ts

```diff
@@ -0,0 +1,22 @@
+export enum CartState {
+  START = 'start',
+  PROCESSING = 'processing',
+  SUCCESS = 'success',
+  FAIL = 'fail',
+}
+
+export interface TaxAmount {
+  title: string;
+  amount: number;
+}
+
+export interface Invoice {
+  totalAmount: number;
+  taxAmounts: [TaxAmount];
+}
+
+export interface Subscription {
+  pageConfigId: string;
+  previousInvoice: Invoice;
+  nextInvoice: Invoice;
+}
```

Comment:
Currently planning on replacing these with their own `ObjectType()` and `*.model.ts` files.